vault/keyring.go

```diff
@@ -253,9 +253,12 @@ func (c KeyRotationConfig) Clone() KeyRotationConfig {
 }
 
 func (c *KeyRotationConfig) Sanitize() {
-	if c.MaxOperations == 0 || c.MaxOperations > absoluteOperationMaximum || c.MaxOperations < absoluteOperationMinimum {
+	if c.MaxOperations == 0 || c.MaxOperations > absoluteOperationMaximum {
 		c.MaxOperations = absoluteOperationMaximum
 	}
+	if c.MaxOperations < absoluteOperationMinimum {
+		c.MaxOperations = absoluteOperationMinimum
```

Comment:
Could we just skip this statement, so c.MaxOperations remains the same as it was previously?

Reply:
no, this is making sure it's never below the min.

Reply:
oh, in the API call we could, yes.

Reply:
Updated